pkg/solana/client/client.go

```diff
@@ -331,6 +333,19 @@ func (c *Client) GetLatestBlock(ctx context.Context) (*rpc.GetBlockResult, error
 	return v.(*rpc.GetBlockResult), err
 }
 
+// GetLatestBlockHeight returns the latest block height of the node based on the configured commitment type
```

Comment:
this is out of scope of the reorg, but pushing it here so we don't need to make a separate merge: [smartcontract-it.atlassian.net/browse/NONEVM-1015](https://smartcontract-it.atlassian.net/browse/NONEVM-1015)